### PR TITLE
SW-478 link to laser head what is new page

### PR DESCRIPTION
**Pull request by khaledsherkawi**

### CODE REVIEW

File: octoprint_mrbeam/templates/laserhead_changed.jinja2 [hunk 1]
```diff
@@ -9,6 +9,7 @@
             <h4> {{ _("Your new Mr Beam dreamcut [S] Laserhead has been successfully installed.") }}</h4>
             <p> {{ _("With this Laserhead you can work now even faster.") }}</p>
             <p> {{ _("This means that we also update the default material settings to the new laserhead as long it is installed to provide you the best possible support. However, we do not update your custom materials, this will need your personal testing and finetuning.") }}</p>
+            <p> {{ _("Read about what changes you can expect with a new and improved Laserhead Mr Beam II dreamcut [S].") }} <a target="_blank" href="{{ _("https://support.mr-beam.org/en/support/solutions/articles/43000640531") }}"> » {{ _("Read more") }} </a></p>
```
Comment (amrsoll) [inline octoprint_mrbeam/templates/laserhead_changed.jinja2]: 1. 
> Read about changes

Or 

> Read about the changes

2. Funky name "Laserhead Mr Beam II dreamcut [S]"

Comment (amrsoll) [reply octoprint_mrbeam/templates/laserhead_changed.jinja2]: perhaps "Laserhead for MrBeam II dreamcut [S]" is less of a mouthful.

Comment (amrsoll) [reply octoprint_mrbeam/templates/laserhead_changed.jinja2]: -> with _the_ new and improved

Comment (khaledsherkawi) [reply octoprint_mrbeam/templates/laserhead_changed.jinja2]: fixed

Comment (JFSolution) [reply octoprint_mrbeam/templates/laserhead_changed.jinja2]: But this laserhead is not olny for the dreamcut [S]

**Review by JFSolution — APPROVED** [review]

**Pull request merged**